packages/xo-web/src/common/intl/messages.js

```diff
@@ -860,8 +860,8 @@ const messages = {
   srRescan: 'Rescan all disks',
   srReconnectAll: 'Connect to all hosts',
   srDisconnectAll: 'Disconnect from all hosts',
-  srForget: 'Forget this SR',
-  srsForget: 'Forget SRs',
+  srForget: 'Forget SRs',
+  srsForget: 'Forget {nSrs, number} SR{nSrs, plural, one {} other {s}}?',
```

Comment:
Why the question mark? It means that users will have to type "Forget 3 SRs?" to confirm their action?